Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -272,12 +272,15 @@ public void Run()
                     var data = new List<DataFeedPacket>();
                     foreach (var subscription in Subscriptions)
                     {
-                        var packet = new DataFeedPacket(subscription.Security, subscription.Configuration);
+                        var config = subscription.Configuration;
+                        var packet = new DataFeedPacket(subscription.Security, config);
 
                         // dequeue data that is time stamped at or before this frontier
                         while (subscription.MoveNext() && subscription.Current != null)
                         {
-                            packet.Add(subscription.Current);
+                            var clone = subscription.Current.Clone(subscription.Current.IsFillForward);
+                            clone.Time = clone.Time.RoundDownInTimeZone(config.Increment, config.ExchangeTimeZone, config.DataTimeZone);
+                            packet.Add(clone);
                         }
```

Comment:
Just curious, is this time rounding an additional bug fix (impact on not only custom data) ?

Reply:
The time rounding here is unfortunate, I do agree, but mirrors the backtesting behavior. We depend on rounded times later in the pipe line, specifically when invoking consolidators. The end time is used to determine if a piece of data is in it's subscribed to, native resolution. A later iteration can communicate this information in another way, for now, this is in line with how backtesting does it.